### PR TITLE
Add `isClosedConversation`

### PR DESCRIPTION
**Pull request by yakov116**
Really this should be renamed to `isClosedConversation` but...

### CODE REVIEW

**Comment by yakov116** [comment]
🚀 ?

**Review by fregante — APPROVED** [review]

**Review by yakov116 — COMMENTED** [review]
One line

**Comment by yakov116** [comment]
Um this PR makes GitHub crash 🎉.

Maybe the `:is()` should not be used here?

**Comment by yakov116** [comment]
> Maybe the `:is()` should not be used here?

Yup

**Comment by yakov116** [comment]
Ouch I see why...

File: index.ts [hunk 1]
```diff
@@ -266,7 +266,9 @@ collect.set('isQuickPR', [
 export const isDraftPR = (): boolean => exists('#partial-discussion-header [title="Status: Draft"]');
 export const isOpenPR = (): boolean => exists('#partial-discussion-header [title="Status: Open"], #partial-discussion-header [title="Status: Draft"]');
 export const isMergedPR = (): boolean => exists('#partial-discussion-header [title="Status: Merged"]');
-export const isClosedPR = (): boolean => exists('#partial-discussion-header [title="Status: Closed"], #partial-discussion-header [title="Status: Merged"]');
+export const isClosedConversation = (): boolean => exists('#partial-discussion-header :is([title="Status: Closed"], [title="Status: Merged"], [title="Status: Closed as not planned]")');
```
Comment (fregante) [inline index.ts:269]: Oops

**Pull request merged**